tests/test_by/tests.py

```diff
@@ -0,0 +1,136 @@
+import random
+import string
+
+from django.core.validators import MaxLengthValidator
+from django.test import TestCase
+
+from localflavor.by import forms
+from .forms import BYTestForm
+
+
+class BYLocalFlavorTests(TestCase):
+
+    def setUp(self):
+        self.form = BYTestForm()
+        self.valid_payload = {
+            '': '',
+        }
+
+    def _gen_pass_id(self, is_valid):
```

Comment:
I think it's better to have a few valid and invalid examples like this so that tests will never get a false positive or negative. That said, the chances are pretty low that there will be problems so feel free to keep things like this if this is what you prefer.